config/config.go

```diff
@@ -294,56 +304,79 @@ type AgentConfig struct {
 	BufferDirectory string `toml:"buffer_directory"`
 }
 
-// InputNames returns a list of strings of the configured inputs.
-func (c *Config) InputNames() []string {
-	name := make([]string, 0, len(c.Inputs))
+// getPluginPrintString returns a string representation of the plugin names
+// based on the NewPluginPrintBehaviour flag
+func getPluginPrintString(plugins pluginNames) string {
+	output := PluginNameCounts(plugins)
+	if PrintPluginConfigSource {
+		return output + plugins.String()
+	}
+
+	return output
+}
```

Comment:
Please merge this back to the `*Names()` functions! Stripping this out to save the `if` is not worth it.

Reply:
Done.